content/terraform-enterprise/1.0.x/docs/enterprise/users-teams-organizations/api-tokens.mdx

```diff
@@ -22,13 +22,9 @@ API tokens may belong directly to a user. User tokens are the most flexible toke
 
 API tokens may belong to a specific team. Team API tokens allow access to the workspaces that the team has access to, without being tied to any specific user.
 
-Navigate to the **Organization Settings > API Tokens > Team Tokens** tab to manage API tokens for a team or create new team tokens.
+Navigate to the **Organization settings > API Tokens > Team Token** tab to manage API tokens for a team or create new team tokens.
 
-Teams can have multiple valid tokens at a time, so long as the tokens' descriptions are unique within the context of the given team. A token without a description is considered a legacy token, and only one legacy token can exist at a given time.
-
-The [legacy API](/terraform/enterprise/api-docs/team-tokens#legacy-team-tokens-api-reference) uses a legacy token, generating a new legacy token invalidates the previous token.
-
-The [current team token API](/terraform/enterprise/api-docs/team-tokens#team-tokens-api-reference) supports multiple team tokens, and adding new tokens does not invalidate older ones.
+Each team can have **one** valid API token at a time. When a token is regenerated, the previous token immediately becomes invalid.
```

Comment:
```suggestion
Each team can have one valid API token at a time. When a token is regenerated, the previous token immediately becomes invalid.
```
Per the [style guide](https://github.com/hashicorp/web-unified-docs/blob/main/docs/style-guide/general/grammar.md#do-not-use-punctuation-or-text-formatting-to-add-semantic-emphasis), we should only use bold for UI elements and introducing new terms.